Files/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -118,34 +149,39 @@
                                         </Setter.Value>
                                     </Setter>
                                 </Style>
-                            </GridView.ItemContainerStyle>
-                            <GridView.ItemTemplate>
+                            </ListView.ItemContainerStyle>
+                            <ListView.ItemTemplate>
                                 <DataTemplate>
                                     <Border>
-                                        <Grid ColumnSpacing="12">
+                                        <Grid ColumnSpacing="12" RowSpacing="15">
```

Comment:
```suggestion
                                        <Grid ColumnSpacing="12" RowSpacing="16">
```
Keep in mind, these numbers should be divisible by 4.

Reply:
Aaaaaaaaaaaa

This is still wip 😉